ext/pdo/tests/pdo_017.phpt

```diff
@@ -12,6 +12,7 @@ PDOTest::skip();
 $db = PDOTest::factory();
```

Comment:
```suggestion
if (str_starts_with(getenv('PDOTEST_DSN'), "firebird")) die('xfail firebird driver does not behave as expected');
$db = PDOTest::factory();
```

Maybe just skip the test for now on Firebird, as it might be a bug in the driver... (this skip is taken from pdo039.phpt)